product_search/product_search.py

```diff
@@ -0,0 +1,42 @@
+"""Program that returns inventory details based for an inventory item"""
+
+
+def validate_item(item_to_search, data):
+    """Verify that the product exists: return True if it exists; return False if it doesn't"""
+
+    # I want to iterate thru ALL items (not just the first one) so I move the return False outside of the loop
+    for item in data["products"]:
+        if item["name"] == item_to_search:
+            return True
+    return False
+
+
+# IF the item is in the json file, then I need to extrac the dictionary. Pass this diction ary to _style_dictionary fucntion
+def inventory_details_for_user_item(item, data):
```

Comment:
Nice function!

Reply:
I'd personally flip the parameter order though, as `data` is higher level than `item`.